src/python/pants/backend/jvm/tasks/jvm_compile/BUILD

```diff
@@ -101,6 +102,31 @@ python_library(
   sources = ['anonymizer.py'],
 )
 
+python_library(
+  name = 'javac',
+  sources = globs('javac/*.py'),
```

Comment:
The convention in this file of defining targets outside of their directory is unfortunate... would recommend defining this target in that directory instead.